epidose/device/util.sh

```diff
@@ -186,6 +189,22 @@ get_new_filter()
   fi
 }
 
+# Check and update device if necessary
+# preconditions: WiFi should be turned on
+check_for_updates()
+{
+  log "Checking for updates"
+  if err=$(curl --silent --show-error --fail --output "$UPDATE.new" \
+    "$SERVER_URL/update" 2>&1) ; then
+    # Atomically replace existing update script with new one
```

Comment:
I don't think atomic update is needed, because nobody else is accessing the file, right?

Reply:
ok!